src/selectors_vm/parser.rs

```diff
@@ -33,7 +33,7 @@ pub enum PseudoElementStub {}
 
 impl ToCss for PseudoElementStub {
     fn to_css<W: fmt::Write>(&self, _dest: &mut W) -> fmt::Result {
-        match *self {}
+        unimplemented!()
```

Comment:
Can you revert these `unimplemented!()`

Before these were proving that it was impossible to panic. Now it looks like they can panic (of of course they can't because this is unreachable), and it looks like it's unimplemented when that's not the case.

Reply:
Previous code triggers a warning that it is UB to dereference an uninhabited type.

Reply:
Oh, interesting. In that case can we make it `unreachable!()` that represents what is it a bit better.